packages/terra-form-select/package.json

```diff
@@ -31,6 +31,7 @@
     "classnames": "^2.2.5",
     "keycode-js": "^2.0.1",
     "lodash.uniqueid": "^4.0.1",
+    "mutationobserver-shim": "0.3.3",
```

Comment:
Is there a reason why using 0.3.3 instead of the latest 0.3.7?

Reply:
Let's keep them all consistent with what we're already using for now.  We can log separate issues to update this later.